frontend/src/pages/modelServing/screens/projects/ServingRuntimeModal/ServingRuntimeTokenSection.tsx

```diff
@@ -11,12 +11,17 @@ import {
 import { PlusCircleIcon } from '@patternfly/react-icons';
 import IndentSection from '~/pages/projects/components/IndentSection';
 import { UpdateObjectAtPropAndValue } from '~/pages/projects/types';
-import { CreatingServingRuntimeObject } from '~/pages/modelServing/screens/types';
+import {
+  CreatingInferenceServiceObject,
+  CreatingServingRuntimeObject,
+} from '~/pages/modelServing/screens/types';
 import ServingRuntimeTokenInput from './ServingRuntimeTokenInput';
 
 type ServingRuntimeTokenSectionProps = {
-  data: CreatingServingRuntimeObject;
-  setData: UpdateObjectAtPropAndValue<CreatingServingRuntimeObject>;
+  data: CreatingServingRuntimeObject | CreatingInferenceServiceObject;
+  setData:
+    | UpdateObjectAtPropAndValue<CreatingServingRuntimeObject>
+    | UpdateObjectAtPropAndValue<CreatingInferenceServiceObject>;
```

Comment:
nit:
```suggestion
  setData: UpdateObjectAtPropAndValue<CreatingServingRuntimeObject | CreatingInferenceServiceObject>;
```

Reply:
Mmmmm I don't think we can do that, since they are not technically the same, meaning that if i pass then data function that it's just `UpdateObjectAtPropAndValue<CreatingServingRuntimeObject>` it will fail since the type is not a superset of `setData: UpdateObjectAtPropAndValue<CreatingServingRuntimeObject | CreatingInferenceServiceObject>;`?